Website/index.js

```diff
@@ -0,0 +1,472 @@
+function show_personalised() {
+    $("#updated").text(jsonData["updated"]);
+    $("#date2").remove();
+
+    $("#personalised").empty();
+
+    function DaysOfWeek(day) {
+        $(Object.keys(jsonData["weeks"])).each(function (i, v) {
+
+            $(jsonData["weeks"][v]).each(function (b, value) {
+                var a = new Date(value);
+
+                if (String(day) == String(a)) {
+                    week = jsonData["weeks"][v]
+                }
+
+            });
+
+        });
+    }
+
+    var day = new Date(String($("#start").val()));
+    DaysOfWeek(day);
+    var week;
+
+    var per_dict = {
+        0: {
+            "id": "day_1",
+            "h4": "Monday",
+        },
+        1: {
+            "id": "day_2",
+            "h4": "Tuesday",
+        },
+        2: {
+            "id": "day_3",
+            "h4": "Wednesday",
+        },
+        3: {
+            "id": "day_4",
+            "h4": "Thursday",
+        },
+        4: {
+            "id": "day_5",
+            "h4": "Friday"
+        }
+    }
+
+    var recess_badge = {
+        1: '<span class="badge bg-primary mb-1">First Recess (9:10 - 9:45)</span>',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8386818)
```suggestion
        1: "<span class=\"badge bg-primary mb-1\">First Recess (9:10 - 9:45)</span>",
```